### PR TITLE
Mod.

### PR DESCRIPTION
**Pull request by Tranvanlap95**
react-cropper/react-cropper

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Review by AfirstS — CHANGES_REQUESTED** [review]
Обновление и запуск, 👑🎷🎺📻📀 @AfirstSℹ️

**Comment by hariharanep4** [comment]
I could not install dependencies to run this project locally, can anyone please guide me how to do that.
Thank you in advance.

**Comment by Tranvanlap95** [comment]
Yes i subscribe changes

Vào 0:17, Th 5, 19 thg 8, 2021 Nick Holden ***@***.***> đã
viết:

**Comment by Witte1979** [comment]
Stop

Op wo 18 aug. 2021 om 19:56 schreef Tranvanlap95 ***@***.***>

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+react-cropper/react-cropper---
```
Comment (Anoos97) [inline index.html:1]: [dumpstate-stats.txt](https://github.com/github/opensource.guide/files/7024787/dumpstate-stats.txt)

- [ ] 

- [ ]

**Review by Anoos97 — COMMENTED** [review]
[dumpstate-stats.txt](https://github.com/github/opensource.guide/files/7024789/dumpstate-stats.txt)
index.html

**Pull request closed without merging**